### PR TITLE
Line 47 updated.

### PR DESCRIPTION
**Pull request by Jagatjot**
**4. Run the Ruby client subscriber with queue group name.** updated with the correct command ruby queue_sub.rb foo my-queue &

### CODE REVIEW

File: content/documentation/tutorials/nats-queueing.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ node node-sub.js foo my-queue
 **4. Run the Ruby client subscriber with queue group name.**
 
 ```
-nats-sub foo my-queue &
+ruby queue_sub.rb foo my-queue &
```
Comment (wallyqs) [inline content/documentation/tutorials/nats-queueing.md]: It should actually be this here instead:
```
gem install nats
nats-queue foo my-queue &
```

Comment (Jagatjot) [reply content/documentation/tutorials/nats-queueing.md]: ruby queue_sub.rb foo my-queue & command gives the below output:

[root@NATS examples]# ruby queue_sub.rb foo my-queue &
[2] 23243
[root@NATS examples]# Listening on [foo], queue group [my-queue]

Comment (Jagatjot) [reply content/documentation/tutorials/nats-queueing.md]: while in nats-sub foo my-queue & command the client only listens on a subject foo without entering any queue group as follows:

[root@NATS examples]# nats-sub foo my-queue &
[3] 23281
[root@NATS examples]# Listening on [foo]

Comment (Jagatjot) [reply content/documentation/tutorials/nats-queueing.md]: uby queue_sub.rb foo my-queue & command gives the below output:

[root@NATS examples]# ruby queue_sub.rb foo my-queue &
[2] 23243
[root@NATS examples]# Listening on [foo], queue group [my-queue]

Comment (Jagatjot) [reply content/documentation/tutorials/nats-queueing.md]: while in nats-sub foo my-queue & command the client only listens on a subject foo without entering any queue group as follows:

[root@NATS examples]# nats-sub foo my-queue &
[3] 23281
[root@NATS examples]# Listening on [foo]

**Comment by Jagatjot** [comment]
Correct so it should change either to ruby queue_sub.rb foo my-queue & or nats-queue foo my-queue & instead of nats-sub foo my-queue & which is currently mentioned in the document.

**Comment by wallyqs** [comment]
agree, I'd say we change to nats-queue since we that is distributed with the gem and in the PATH. Thanks!

**Comment by Jagatjot** [comment]
@wallyqs Thanks I have updated it with nats-queue foo my-queue

**Pull request merged**